sys/fmt/fmt.c

```diff
@@ -118,6 +123,25 @@ size_t fmt_s32_dec(char *out, int32_t val)
     return fmt_u32_dec(out, val) + negative;
 }
 
+size_t fmt_s16_dec(char *out, int16_t val)
```

Comment:
True... 

Back to the question: I don't care if it is inlined or not, I just copied what was there already... I would go with this now, feel free to PR a fix with inlining some of the functions.